src/components/ThemeStylesProvider.tsx

```diff
@@ -1,17 +1,19 @@
 import React, {useMemo} from 'react';
 import useTheme from '@hooks/useTheme';
-import stylesGenerator from '@styles/index';
+// eslint-disable-next-line no-restricted-imports
+import styles from '@styles/index';
 import ThemeStylesContext from '@styles/theme/context/ThemeStylesContext';
+// eslint-disable-next-line no-restricted-imports
 import createStyleUtils from '@styles/utils';
 
 type ThemeStylesProviderProps = React.PropsWithChildren;
 
 function ThemeStylesProvider({children}: ThemeStylesProviderProps) {
     const theme = useTheme();
 
-    const styles = useMemo(() => stylesGenerator(theme), [theme]);
-    const StyleUtils = useMemo(() => createStyleUtils(theme, styles), [theme, styles]);
-    const contextValue = useMemo(() => ({styles, StyleUtils}), [styles, StyleUtils]);
+    const themeStyles = useMemo(() => styles(theme), [theme]);
+    const StyleUtils = useMemo(() => createStyleUtils(theme, themeStyles), [theme, themeStyles]);
```

Comment:
could you remind me of the reason for this rename?

Reply:
The outwards facing context variables haven't been renamed. This is just for consistency reasons